src/adapters/authAdapter.ts

```diff
@@ -5,19 +5,40 @@ type LoginAuthType = {
   password: string;
 };
 
+/* eslint-disable camelcase */
+export const OauthParams = {
+  client_id: process.env.REACT_APP_API_CLIENT_ID,
+  client_secret: process.env.REACT_APP_API_CLIENT_SECRET,
+};
+/* eslint-enable camelcase */
 class AuthAdapter extends BaseAdapter {
-  static login(params: LoginAuthType) {
+  static loginWithEmailPassword(authParams: LoginAuthType) {
     /* eslint-disable camelcase */
     const requestParams = {
-      ...params,
+      ...OauthParams,
+      ...authParams,
       grant_type: 'password',
-      client_id: process.env.REACT_APP_API_CLIENT_ID,
-      client_secret: process.env.REACT_APP_API_CLIENT_SECRET,
     };
     /* eslint-enable camelcase */
 
     return this.prototype.postRequest('oauth/token', { data: requestParams });
   }
+
+  static loginWithRefreshToken(refreshToken: string) {
```

Comment:
Should we rename this function? Because user won't login after running this function but get the new access token.
```suggestion
  static refreshAccessToken(refreshToken: string) {
  static refreshUserToken(refreshToken: string) {
```

Reply:
I named it this because the only difference between the login endpoint is we are sending a `refresh_token` instead of `email/password`, otherwise endpoint and response are identical.

I also deemed "login" just to be storing token in localstorage which is what both do.

Do you still think it's better to change it?

Reply:
In my opinion, the user needs to log in with their email/password first before they get a refresh token. The user can't initially log in with a refresh token. So to get a new access token, it sounds like the user refreshes the token, not re-login. 
Anyway I don't have a strong opinion about this one, I am fine with both name 👍

Reply:
I didn't think of that - you're right there's no way to get a refresh_token without logging in first, so I will change the name 👍  @hanam1ni

Reply:
I will add it in the survey list PR https://github.com/liamstevens111/react-survey-ic/pull/41 as it's all moved to just `surveyAdapter` anyway

Reply:
As authAdapter changed to all be inside surveyAdapter in survey list PR i have incldued it here in that PR:
https://github.com/liamstevens111/react-survey-ic/pull/41/commits/66a6f8c97d87b85ddb05e846a426319ca2ede2fc

@hanam1ni fyi